roles/pulp_api/tasks/main.yml

```diff
@@ -11,4 +12,11 @@
       become: true
       notify: Restart pulpcore-api.service
 
+    - name: Set the pulpcore-api service state
```

Comment:
I faced the following when trying the branch

```
TASK [pulp_api : Set the pulpcore-api service state] ***********************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "failure 1 during daemon-reload: Failed to execute operation: Interactive authentication required.\n"}
```

Reply:
Fixed, I forgot `become: true`